### ISSUE TITLE
Distorted Text In Chat

### ISSUE THREAD

**Issue by JustAlittleWolf** [sp614x/optifine#2791]
When using optifine and a special character is in front of some text in chat it gets distorted. I have tried newer and older versions of Optifine but nothing works. As soon as i stop using Optifine everything is back to normal. Note that the issue only appears after the special character. The version is Optifine OptiFine 1.8.9 HD U I7
With Optifine:
![Optifine](https://user-images.githubusercontent.com/54244277/63228016-72444f00-c1ed-11e9-91f1-58179822f23a.png)
Without Optifine:
![Default](https://user-images.githubusercontent.com/54244277/63228018-76706c80-c1ed-11e9-986f-a7cbfd358f8c.png)

**Comment by sp614x** [comment]
The text seems OK, only the special char has no shadow.
What is distorted?

**Comment by JustAlittleWolf** [comment]
Sorry if the issue wasn't clear, here's a close look:
![opti](https://user-images.githubusercontent.com/54244277/64060755-553a5380-cbd1-11e9-8876-63035e845878.png)

**Comment by JustAlittleWolf** [comment]
So I've come back to this problem and it turns out this issue only happens on the GUI scale i play with, which is Large, and not on the other ones like Auto, Small and Normal. I have a 1920x1080 display and changing the window size doesnt matter neither does fullscreen on or off.

I have also tried other approaches to this problem, such as deleting the .minecraft folder and reinstalling the game and change the font with a texture pack or using a different version of optifine or using a client with optifine embedded.

This Issue is particularly annoying when playing on Mulitplayer servers as, hypixel for example, uses these special characters as a display of the rank at the beginning of a chat message thus distorting the entire text afterwards. My friend also has this problem (although he doesn't bother and most people don't) and if you're watching a youtube video you might also get a glimpse of it.

random youtube video with this issue: https://www.youtube.com/watch?v=BpPRX-Ec9d8&feature=youtu.be&t=49

**Comment by Brottweiler** [comment]
Yeah this is a problem with GUI scales. Chat text usually works fine, but if that line has a special character the font can get distorted. But this is a vanilla issue I think and not Optifine.

**Comment by JustAlittleWolf** [comment]
> 
> 
> Yeah this is a problem with GUI scales. Chat text usually works fine, but if that line has a special character the font can get distorted. But this is a vanilla issue I think and not Optifine.

I have been playing Minecraft since 2015 and this problem only occured around a year ago when i first installed Optifine.

**Comment by JustAlittleWolf** [comment]
Moved on from optifine 👍
